src/bin/cargo/commands/test.rs

```diff
@@ -7,7 +7,7 @@ pub fn cli() -> App {
         // subcommand aliases are handled in aliased_command()
         // .alias("t")
         .setting(AppSettings::TrailingVarArg)
-        .about("Execute all unit and integration tests of a local package")
+        .about("Execute all unit and integration tests and examples of a local package")
```

Comment:
Hm, examples aren't executed or necessarily tested.  Maybe "and build examples"?

Reply:
You're right, thanks.  Done.